tutorials/analyzers.ipynb

```diff
@@ -6,14 +6,115 @@
    "source": [
```

Comment:
<p>I don't see a reason the same code won't work with pydeequ 1.4.0/Spark 3.5. I think it might be fine to insert something like "Tested on pydeequ 1.2.0/Spark 3.3. Code should run on all supported pydeequ versions". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/python-deequ/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>